src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj

```diff
@@ -13,9 +14,19 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisCSharpWorkspacesFixedVersion)" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit" Version="$(MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitVersion)" />
   </ItemGroup>
+  <ItemGroup Label="Project References">
+    <!-- Directly reference the Workspaces project so we always test against the latest Roslyn bits -->
```

Comment:
This change is needed to ensure that we always run Code style tests against the latest Roslyn Compiler and Workspaces bits.